kobo/rcS

```diff
@@ -83,8 +83,38 @@ ulimit -c unlimited
 # workaround for Kobo Clara HD: the touch screen doesn't respond
 # and the initial screen render is blank if we are too fast
 # delay should not cause any problem and might help other models
-usleep 100000
-
+sleep 1
```

Comment:
Why?

Reply:
With 100000us (0.1s) I had a few times when the Clara HD started with no touch screen.  I felt it was a harmless increase to make it 1s and I have not yet had a single failure.  Sad that the Kobo is that touchy!

Reply:
So... this has nothing to do with OTG support?
Such a change needs to be a separate commit with an explanatory commit message.

Reply:
OK  will remove the change and (possibly) submit later.  Extra code in rcS may add enough delay.

Reply:
returned to original